src/translib/common_app.go

```diff
@@ -726,6 +733,44 @@ func checkAndProcessLeafList(existingEntry db.Value, tblRw db.Value, opcode int,
 	return tblRw
 }
 
+func intf_app_translateSubscribe(dbs [db.MaxDB]*db.DB, path string) (*notificationOpts, *notificationInfo, error) {
+	appDB := dbs[db.ApplDB]
+	pathInfo := NewPathInfo(path)
+	notifInfo := notificationInfo{dbno: db.ApplDB}
+	notSupported := tlerr.NotSupportedError{Format: "Subscribe not supported", Path: path}
+
+	if isSubtreeRequest(pathInfo.Template, "/openconfig-interfaces:interfaces") {
+		if pathInfo.HasSuffix("/interface{}") ||
+			pathInfo.HasSuffix("/config") ||
+			pathInfo.HasSuffix("/state") {
+			log.Errorf("Subscribe not supported for %s!", pathInfo.Template)
+			return nil, nil, notSupported
+		}
+		ifName := pathInfo.Var("name")
+		if len(ifName) == 0 {
+			return nil, nil, errors.New("Empty Interface name")
+		}
+		log.Info("Interface name = ", ifName)
+		ifKey := db.Key{Comp: []string{ifName}}
+		portTblTs := &db.TableSpec{Name: "PORT_TABLE"}
```

Comment:
PORT_TABLE is only for physical interfaces, to make it more generic(subscribe support for other interface types-LAG, VLAN..)  then we will have to find type of interface from ifName and then  look into its table in appDB.

Reply:
@Tejaswi-Goel  - we were just asked to pot the  code from arlo....no new support for now